composer.json

```diff
@@ -25,9 +25,9 @@
     },
     "require": {
         "php": ">=5.5.9",
-        "symfony/framework-bundle": "^3.4",
-        "symfony/twig-bundle": "^3.4",
-        "twig/twig": "^2.0"
```

Comment:
And if we want to support legacy versions I mentioned above - bumping twig to the new major version might be too aggressive. I'd rather allow Twig 2.0 or Twig 3.0, i.e. `^2.0|^3.0`